bench/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "0.0.0",
+  "private": true,
+  "license": "WTFPL",
```

Comment:
I take it the only reason we need this file at all is because we want to install the previously published version of sanctuary-type-classes?